### PR TITLE
Add stop_area, stop_point to forbidden_uri

### PR DESCRIPTION
**Pull request by l-vincent-l**
Wait before merging this PR, it needs some benchmark.

There's no incidence on performances, so it can be merged.

### CODE REVIEW

File: source/routing/tests/raptor_test.cpp [hunk 1]
```diff
@@ -446,6 +446,25 @@ BOOST_AUTO_TEST_CASE(validity_pattern){
 }
 */
 
+
+BOOST_AUTO_TEST_CASE(forbidden_uri){
```
Comment (antoine-de) [inline source/routing/tests/raptor_test.cpp:450]: un p'tit commentaire siouplait

File: source/routing/raptor.cpp [hunk 2]
```diff
@@ -288,7 +288,21 @@ void RAPTOR::set_journey_patterns_valides(uint32_t date, const std::vector<std::
             }
         }
     }
-    journey_patterns_valides &= ~forbidden_journey_patterns;
+    valid_journey_patterns &= ~forbidden_journey_patterns;
+
+    for(const type::JourneyPatternPoint* jpp: data.pt_data->journey_pattern_points) {
```
Comment (antoine-de) [inline source/routing/raptor.cpp]: wouldn't it be better to loop first on the forbidden (then on all jpp), and hence not to have any overhead if no forbidden is given?

**Comment by l-vincent-l** [comment]
@antoine-de I have taken in account your remarks

File: source/routing/raptor.cpp [hunk 3]
```diff
@@ -182,7 +182,7 @@ void RAPTOR::init(Solutions departs,
         if(sp->accessible(required_properties)) {
             for(auto journey_pattern_point : sp->journey_pattern_point_list) {
                 type::idx_t jppidx = journey_pattern_point->idx;
-                if(journey_patterns_valides.test(journey_pattern_point->journey_pattern->idx)) {
+                if(valid_journey_patterns.test(journey_pattern_point->journey_pattern->idx)) {
```
Comment (l-vincent-l) [inline source/routing/raptor.cpp]: I need to add valid_journey_pattern_points

**Pull request merged**